example/newreposecret/go.mod

```diff
@@ -6,4 +6,5 @@ require (
 	github.com/GoKillers/libsodium-go v0.0.0-20171022220152-dd733721c3cb
 	github.com/google/go-github/v33 v33.0.0
 	golang.org/x/oauth2 v0.0.0-20200902213428-5d25da1a8d43
+	gopkg.in/yaml.v2 v2.3.0
```

Comment:
I think it would be better to add a "// indirect" comment at the end of this line.

It's added automatically if you use `go get` to select this version:

```diff
$ go get -d gopkg.in/yaml.v2@v2.3.0
go get: upgraded gopkg.in/yaml.v2 v2.2.2 => v2.3.0
$ git diff -- go.mod               
diff --git a/example/newreposecret/go.mod b/example/newreposecret/go.mod
index 5d05d95..e902fbf 100644
--- a/example/newreposecret/go.mod
+++ b/example/newreposecret/go.mod
@@ -6,4 +6,5 @@ require (
        github.com/GoKillers/libsodium-go v0.0.0-20171022220152-dd733721c3cb
        github.com/google/go-github/v33 v33.0.0
        golang.org/x/oauth2 v0.0.0-20200902213428-5d25da1a8d43
+       gopkg.in/yaml.v2 v2.3.0 // indirect
 )
```

Reply:
Sounds good.
Done.